miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppStorage.kt

```diff
@@ -43,6 +49,33 @@ internal class MiniAppStorage(
     @VisibleForTesting
     fun getFileName(file: String) = urlToFileInfoParser.getFileName(file)
 
-    fun getSavePathForApp(appId: String, versionId: String) =
-        "${basePath.path}/$SUB_DIR_MINIAPP/$appId/$versionId"
+    @VisibleForTesting
+    internal fun getBasePathHostApp() = basePath.path
+
+    @VisibleForTesting
+    internal fun getRootPathMiniApp() = "${getBasePathHostApp()}/$SUB_DIR_MINIAPP/"
+
+    @VisibleForTesting
+    internal fun getPathMiniApp(appId: String) = "${getRootPathMiniApp()}/$appId/"
+
+    fun getPathMiniAppVersion(appId: String, versionId: String) = "${getPathMiniApp(appId)}$versionId"
```

Comment:
As we spoke, all the functions aren't required to be marked for testing. 
If you only test the consumer function, that should suffice. See if the following looks better.

```
    // Documentation goes here
    private val hostAppBasePath = basePath.path

    // Documentation goes here
    private val miniAppBasePath
        get() = "$hostAppBasePath/$SUB_DIR_MINIAPP/"

    // Documentation goes here
    private val miniAppPath = { appId: String -> "${miniAppBasePath}$appId/" }

    @VisibleForTesting
    // Documentation goes here
    fun getMiniAppVersionPath(appId: String, versionId: String) =
        "${miniAppPath(appId)}$versionId"

Reply:
Thanks!
I still keep `miniAppPath` as function for testing purpose and lamda expression functionality is not different from the function itself.